### DISCUSSION TITLE
Avoir des informations spécifiques pour les retraités qui crée une activité

### DISCUSSION THREAD

**Discussion by johangirod** [betagouv/mon-entreprise#2766] [Idées]
C'est une typologie d'utilisateur qui n'a pas les mêmes besoins que les autres en termes de couverture sociale. Par ailleurs, il y a certaines exonérations (?) liées au statut de retraité.

Enfin, s'il n'y a pas d'exonération, il faudrait le préciser.

Connexe #2161

**Comment by supersuperflux** [comment]
Je rebondis sur ce point, à mon sens ça serait pas mal d'ajouter la réflexion lorsqu'on bossera sur la page d'accueil : actuellement, on met en avant les populations salariées et indépendantes, mais demain, on pourrait songer à avoir plus de granularité et à faire des simulateurs dédiés pour cette population-là au lieu faire un macro-simulateur qui gère tous les cas. Ça permettrait aussi de mettre en avant l'idée des sous-simulateurs : au lieu de gérer le cas, on renvoie vers le simulateur dédié.
